File: app/src/main/java/com/ems/dingdong/model/EWalletRemoveDataRequest.java
package com.ems.dingdong.model;

import com.ems.dingdong.model.response.EWalletDataResponse;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class EWalletRemoveDataRequest {
    @SerializedName("Code")
    @Expose
    private String code;
    @SerializedName("EWalletRemoveRequest")
    List<EWalletDataResponse> eWalletRemoveRequest;
    @SerializedName("Signature")
    String signature;

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public List<EWalletDataResponse> geteWalletRemoveRequest() {
        return eWalletRemoveRequest;
    }

    public void seteWalletRemoveRequest(List<EWalletDataResponse> eWalletRemoveRequest) {
        this.eWalletRemoveRequest = eWalletRemoveRequest;
    }

    public String getSignature() {
        return signature;
    }

    public void setSignature(String signature) {
        this.signature = signature;
    }
}

File: app/src/main/java/com/ems/dingdong/model/request/EWalletRemoveDataRequest.kt
package com.ems.dingdong.model.request

import com.google.gson.annotations.SerializedName

data class EWalletRemoveDataRequest(
    @SerializedName("LadingCode")
    val ladingCode:String?,
    @SerializedName("RetRefNumber")
    val retRefNumber :String?,
    @SerializedName("RemoveBy")
    val removeBy :Long?,
    @SerializedName("POCode")
    val pOCode :String?,
    @SerializedName("ReasonType")
    val reasonType :String?,
    @SerializedName("ReasonText")
    val reasonText :String?,
    @SerializedName("ServiceCode")
    val serviceCode :String?

)